packages/ra-core/src/i18n/TranslatableContext.ts

```diff
@@ -5,8 +5,6 @@ export const TranslatableContext = createContext<
 >(undefined);
 
 export interface TranslatableContextValue {
-    getLabel: GetTranslatableLabel;
```

Comment:
The two types lines 13 & 14 also need removing

Reply:
Removed